packages/models-library/src/models_library/projects.py

```diff
@@ -87,6 +88,14 @@ class ProjectAtDB(ProjectCommons):
         False, description="Defines if a study is available publicly"
     )
 
+    boot_options: Optional[Dict[NodeIDStr, Dict[EnvVarKey, str]]] = Field(
```

Comment:
it would be nice to have some alias here following our conventions to make this more readable

Reply:
and actually is it optional in the DB?